src/NzbDrone.Core/Extras/Files/ExtraFileManager.cs

```diff
@@ -19,7 +20,8 @@ public interface IManageExtraFiles
         IEnumerable<ExtraFile> CreateAfterEpisodeImport(Series series, EpisodeFile episodeFile);
         IEnumerable<ExtraFile> CreateAfterEpisodeFolder(Series series, string seriesFolder, string seasonFolder);
         IEnumerable<ExtraFile> MoveFilesAfterRename(Series series, List<EpisodeFile> episodeFiles);
-        ExtraFile Import(Series series, EpisodeFile episodeFile, string path, string extension, bool readOnly);
+        bool HandleFileImport(LocalEpisode localEpisode, EpisodeFile episodeFile, string path, string extension, bool readOnly);
```

Comment:
I'm not sure this name makes the most sense for this method, I think it's too easily confused with a function that actually handles the import. `CanImportFile` better indicates that it can be handled, but open to other ideas.